release-1.8/release_notes_draft.md

```diff
@@ -234,6 +234,13 @@ across the system. Here's the release [scalability validation report].
   - In v1.9, the default will be `""`, which means no built-in cloud provider extension will be enabled by default.
   - If you want to use an out-of-tree cloud provider in either version, you should use `--cloud-provider=external`
   - [PR #51312](https://github.com/kubernetes/kubernetes/pull/51312) and [announcement](https://groups.google.com/forum/#!topic/kubernetes-dev/UAxwa2inbTA)
+- The `PersistentVolumeLabel` admission controller in the API Server is deprecated
+  - The replacement is running a cloud-specific out-of-tree controller-manager (often referred to as `cloud-controller-manager`) with the
+    `PersistentVolumeLabel` controller enabled. This new controller loop does exactly what this admission controller used to do.
+  - Please stop using this admission controller in your configuration files and scripts now **unless** you're dependent on the GCE and AWS
```

Comment:
Do not use this admission controller in the configuration files and scripts unless you are dependent on the GCE and AWS platforms.